public/API/API_GetGameExtended.php

```diff
@@ -81,7 +82,8 @@
     'Publisher' => $game->Publisher,
     'Developer' => $game->Developer,
     'Genre' => $game->Genre,
-    'Released' => $game->Released,
+    'Released' => $game->released_at?->format('Y-m-d H:i:s'),
```

Comment:
see comment about including hour/minute/second: https://github.com/RetroAchievements/RAWeb/pull/2588#discussion_r1719026370

Reply:
Agreed. Set to YYYY-MM-DD in latest.